content/templates.xqm

```diff
@@ -43,13 +45,17 @@ declare variable $templates:ATTR_DATA_TEMPLATE := "data-template";
  :
  : <pre>function($functionName as xs:string, $arity as xs:int) { function-lookup(xs:QName($functionName), $arity) }</pre>
  :
- : @param $content the sequence of nodes which will be processed
+ : @param $template the template that will be processed
  : @param $resolver a function which takes a name and returns a function with that name
- : @param $model a sequence of items which will be passed to all called template functions. Use this to pass
+ : @param $model a map which will be passed to all called template functions. Use this to pass
  : information between templating instructions.
 :)
-declare function templates:apply($content as node()+, $resolver as function(xs:string, xs:int) as item()?, $model as map(*)?) {
-    templates:apply($content, $resolver, $model, ())
+declare function templates:apply(
+    $template as node(),
```

Comment:
Allowing for only one (main) template to be rendered at the start allowed for some aggressive optimisations. This is also a breaking change

Reply:
Though I suspect __no one__ has ever used templates:apply to render multiple nodes at the same time.